src/styles/main.scss

```diff
@@ -1,3 +1,245 @@
+  display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: space-between;
+}
+
+.card {
+  position: relative;
+  box-sizing: content-box;
+  font-family: "Roboto", Arial, Helvetica, sans-serif;
+  font-style: normal;
+  font-weight: 500;
+  font-size: 12px;
+  line-height: 18px;
+
+  width: 198px;
+  height: 406px;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+
+  &:nth-child(-n + 4) {
+    margin-top: 50px;
+  }
+
+  &:nth-last-child(-n + 4) {
+    align-self: flex-end;
+  }
+
+  &__product-photo {
+    position: absolute;
+    left: 19px;
+    top: 32px;
+    width: 160px;
+    height: 134px;
+  }
+
+  &__product-title {
+    text-decoration: none;
+    position: absolute;
+    width: 166px;
+    height: 36px;
+    left: 16px;
+    top: 206px;
+    color: $Main-accent;
+  }
+
+  &__product-code {
+    position: absolute;
+    width: 98px;
+    height: 14px;
+    top: 246px;
+    left: 16px;
+
+    font-weight: 400;
+    font-size: 10px;
+    line-height: 14px;
+    color: $Secondary;
+  }
+
+  &__review {
+    position: absolute;
+    display: flex;
+    left: 16px;
+    top: 276px;
+    height: 16px;
+    width: 166px;
+    font-weight: 400;
+    font-size: 10px;
+    color: $Main-accent;
+    justify-content: space-between;
+    flex-wrap: nowrap;
+  }
+
+  &__item {
+    width: 53px;
+    text-align: right;
+  }
+
+  &__price {
+    position: absolute;
+    left: 16px;
+    top: 316px;
+    width: 31px;
+    height: 18px;
+    color: $Secondary;
+    font-family: "Roboto", Arial, Helvetica, sans-serif;
+    font-weight: 400;
+  }
+
+  &__amount {
+    position: absolute;
+    text-align: right;
+    right: 16px;
+    top: 316px;
+    width: 50px;
+    height: 18px;
+    font-family: "Roboto", Arial, Helvetica, sans-serif;
+    font-size: 16px;
+    font-weight: 700;
+    color: $Main-accent;
+  }
+
+  &__button {
+    box-sizing: border-box;
+    position: absolute;
+    text-transform: uppercase;
+    height: 40px;
+    width: 166px;
+    left: 16px;
+    bottom: 16px;
+    border: 1px solid $Blue-accent;
+    border-radius: 5px ;
+    background-color: $Blue-accent;
+    color: #fff;
+    text-align: center;
+    line-height: 38px;
+    font-weight: 700;
+    text-decoration: none;
+    font-size: 14px;
+  }
+
+  &__button:hover {
+    background-color: #fff;
+    color: $Blue-accent;
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: flex-start;
+  // position: absolute;
+  left: 16px;
+  top: 276px;
+  height: 16px;
+  width: 96px;
+
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url(../images/star.svg);
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+    line-height: 16px;
+  }
+
+  @for $index from 0 through 5 {
+    &--#{$index} > &__star:nth-child(-n + #{$index}) {
+      height: 16px;
+      background-image: url(../images/star-active.svg);
+      background-repeat: no-repeat;
+      background-position: center;
```

Comment:
loops should contain only properties that change, in this case only background-image